docs/ReleaseNotes.md

```diff
@@ -0,0 +1,45 @@
+# Release Notes
+This document contains information about the releases of this crate.
+
+## Version 0.1.0
```

Comment:
doesnt this go into a changelog? this doesnt seem like the right document for this.

Reply:
actually maybe this can just go into the readme?

Reply:
Here we describe all things that are added in this version. We also could discuss bug fixes here in the future. I now only left the added features. And moved the rest to a file CONTRIBUTING.md. Where in I also moved the style decisions.

Reply:
If that is it then we should have a doc called CHANGELOG.md